src/utils.jl

```diff
@@ -13,6 +13,35 @@ function escapeHTML(i::String)
     return o
 end
 
+macro retry(expr)
+    :(@retry 2 $(esc(expr)))
+end
+
+macro retry(N, expr)
+    :(@retryif Any $N $(esc(expr)))
+end
+
+macro retryif(cond, expr)
+    :(@retryif $(esc(cond)) 2 $(esc(expr)))
+end
+
+macro retryif(cond, N, expr)
```

Comment:
I definitely took inspiration from Retry.jl for these macros. Yeah, I think they'd live well over there. I figure we can let the dust settle on all this new error-handling logic to make sure everything goes smoothly.